### PR TITLE
[close #213] TiKV-CDC Chinese manual

### LINKED ISSUE #213: cdc: Request Chinese manual for China customers.
## Feature Request

**Is your feature request related to a problem? Please describe:**

**Describe the feature you'd like:**

Request Chinese manual for China customers.

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by pingyu**
Signed-off-by: pingyu <yuping@pingcap.com>

### What problem does this PR solve?

Issue Number: close #213

Problem Description: **Request Chinese manual for China customers**

### What is changed and how does it work?

Add TiKV-CDC Chinese manual

### Code changes

- No code

### Check List for Tests

This PR has been tested by at least one of the following methods:
- No code

### Side effects

- No side effects

### Related changes

- No related changes

### CODE REVIEW

File: cdc/manual-cn.md [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+2. 根据[模板](https://github.com/tikv/migration/blob/main/cdc/deployments/tikv-cdc/config-templates/scale-out.example.yaml)创建扩容配置文件。
+3. 通过 `tiup cluster scale-out` 扩容 TiKV-CDC 组件（TiUP 扩容可参考 [使用 TiUP 扩容缩容 TiDB 集群](https://docs.pingcap.com/zh/tidb/stable/scale-tidb-using-tiup)）。
+```
+tiup cluster scale-out <cluster-name> scale-out.yaml
+```
+
+### 手工部署
+
+1. 部署两个 TiKV 集群，分别作为上游集群和下游集群。
+2. 启动 TiKV-CDC 集群，包含一个或多个 TiKV-CDC servers。TiKV-CDC server 的启动命令是 `tikv-cdc server --pd <upstream PD endpoints>`.
```
Comment (zeminzhou) [inline cdc/manual-cn.md:25]:
```suggestion
2. 启动 TiKV-CDC 集群，包含一个或多个 TiKV-CDC servers。TiKV-CDC server 的启动命令是 `tikv-cdc server --pd <upstream PD endpoints>` 。
```

File: cdc/manual-cn.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+3. 通过以下命令启动同步任务：`tikv-cdc cli changefeed create --pd <upstream PD endpoints> --sink-uri tikv://<downstream PD endpoints>`
```
Comment (zeminzhou) [inline cdc/manual-cn.md:26]:
```suggestion
3. 通过以下命令启动同步任务：`tikv-cdc cli changefeed create --pd <upstream PD endpoints> --sink-uri tikv://<downstream PD endpoints>`。
```

**Review by zeminzhou — APPROVED** [review]
LGTM~

**Review by haojinming — APPROVED** [review]
LGTM

File: cdc/manual-cn.md (hunk 1 above)
Comment (pingyu) [reply cdc/manual-cn.md]: Fixed, PTAL~

File: cdc/manual-cn.md (hunk 2 above)
Comment (pingyu) [reply cdc/manual-cn.md]: Fixed, PTAL~

**Pull request merged**